addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/ssti/StoredSinkPoint.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.ascanrulesAlpha.ssti;
+
+import java.io.IOException;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpSender;
+import org.zaproxy.addon.commonlib.http.ComparableResponse;
+
+/**
+ * Representation of a location where a given input is reflected after being sent in other location.
+ * This can be used to get the state of the sink and store information about it. The main
+ * information stored is the response to the original request and the response to a reference
+ * request. This information may be useful to compare with the current state of the sink.
+ *
+ * @author DiogoMRSilva (2018)
+ */
+public class StoredSinkPoint implements SinkPoint {
```

Comment:
Some review comments in this class apply to `ReflectedSinkPoint`.

Reply:
Sorry, I'm not clear on this comment.

Reply:
Just saying that some comments done in this class also apply to the other, e.g. change some instance variables to final.